stubs/caldav/caldav/objects.pyi

```diff
@@ -115,6 +115,7 @@ class Calendar(DAVObject):
         todo: bool | None = ...,
         include_completed: bool = ...,
         sort_keys: Sequence[str] = ...,
+        split_expanded: bool = ...,
```

Comment:
I think this also needs to be added to the other overloads.

Reply:
Missed that -- thanks!